packages/core-sdk/.env.example

```diff
@@ -1,2 +1,4 @@
 TEST_WALLET_ADDRESS = 0x0000000000000000000000000000000000000000
-WALLET_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
\ No newline at end of file
+WALLET_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
+JUDGE_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
+WALLET_PRIVATE_KEY_2 = 0x0000000000000000000000000000000000000000
```

Comment:
Can we try to limit the amount of setup up work needed? Instead of relying on requiring two funded test wallets, can we see if we can create the second wallet programmatically and then have wallet 1 transfer enough tokens for gas?

Reply:
I was trying to avoid this because the integration tests are VERY slow, but I did the changes, it's now creating an funding a wallet each time the tests are ran.